runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -196,6 +198,17 @@ default boolean sendNotificationsWhenFocused()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "clientFont",
+		name = "Font",
+		description = "Configure what font is used for the client and runelite added overlays",
+		position = 29
+	)
+	default Font clientFont()
```

Comment:
Is this not the same config as https://github.com/runelite/runelite/blob/1ec40f7692853e907f1afbf130f7e1c6b3f817e9/runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java#L234

Reply:
No. You set the font you want to use in all of runelite with clientFont() config entry. The 3 drop downs below clientFont allow you to change if certain overlays use the small regular or bold version of the chose font in clientFont. interfaceFontType would change the type of the selected font for boost info panel or clue info panel for example.